test/CMakeLists.txt

```diff
@@ -337,11 +344,16 @@ else ()
   add_test(NAME order_polytope_line_intersect COMMAND order_polytope -tc=line_intersect)
   add_test(NAME order_polytope_reflection COMMAND order_polytope -tc=reflection)
   add_test(NAME order_polytope_vec_mult COMMAND order_polytope -tc=vec_mult)
-
-	set(ADDITIONAL_FLAGS "-DSIMD_LEN=0 -march=native -DTIME_KEEPING")
-	set_target_properties(benchmarks_crhmc PROPERTIES COMPILE_FLAGS ${ADDITIONAL_FLAGS})
-	set_target_properties(crhmc_polytope_preparation_test PROPERTIES COMPILE_FLAGS ${ADDITIONAL_FLAGS})
-
+	set(ADDITIONAL_FLAGS "-march=native -DSIMD_LEN=0 -DTIME_KEEPING")
+
+	set_target_properties(benchmarks_crhmc
```

Comment:
This will add the benchmarks in the test-suite. It think we do not want them to be there since they will increase the time without adding to coverage. 

I propose to keep benchmarks commented out for now. Maybe in the future we should collect them in one place.

If there are simple tests in the benchmarks files please add them to unit tests (used for coverage and test the correctness of algorithms and functions ) and leave in benchmarks only the ones needed to benchmark your code.

Reply:
the benchmark is not executed as unit test are often multi-threaded it is simply compiled. But I can  comment out the compilation.